src/components/animation/styledAnimation.js

```diff
@@ -0,0 +1,34 @@
+import styled from "styled-components/macro";
+import { motion } from 'framer-motion';
+
+export const AnimationWrapper = styled.div`
+  width: 36.3%;
```

Comment:
I'll think we'll just adjust percentages based on media queries to get the lower screen resolutions figured out.  This seems fine for now

Reply:
yeah depending on how the rest of the section changes, there will only need to be 2 percentages that change in this section. So that's good!